src/NodeComposerPlugin.php

```diff
@@ -126,4 +126,18 @@ public function onPostUpdate(Event $event)
             }
         }
     }
-}
\ No newline at end of file
+
+    /**
+     * @inheritDoc
+     */
+    public function deactivate(Composer $composer, IOInterface $io)
+    {
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public function uninstall(Composer $composer, IOInterface $io)
```

Comment:
A cleanup would be nice. Like remove all installed node versions.

Reply:
I aggree, but at the moment I would like not implement this by myself. There are many things todo.